sources/packages/backend/apps/queue-consumers/src/processors/schedulers/esdc-integration/ecert-integration/_tests_/ecert-full-time-feedback-integration.scheduler.e2e-spec.ts

```diff
@@ -385,5 +395,83 @@ describe(
         });
       },
     );
+
+    it("Should generate a notification to the ministry when there are ecert feedback errors that block funding for a disbursement.", async () => {
+      // Arrange
+      const notificationMessageType =
+        NotificationMessageType.ECertFeedbackFileErrorNotification;
+      // Remove all the eCert feedback file error notification records before
+      // the run of the test, so that it doesn't interfere with the test data.
+      await db.notification.delete({
+        notificationMessage: {
+          id: notificationMessageType,
```

Comment:
`notificationMessageType` is already a const. I did not see a big improvement if the goal is to increase the readability. IMO we do not need the "alias" in this situation.